libraries/stdlib/src/kotlin/random/Random.kt

```diff
@@ -267,10 +268,14 @@ public abstract class Random {
      *
      * @sample samples.random.Randoms.defaultRandom
      */
-    companion object Default : Random() {
+    companion object Default : Random(), Serializable {
+        private const val serialVersionUID = 0L
```

Comment:
Bad news: this const doesn't work as we intended it to. The reason is that it is compiled as a static field to the `Random` class, and not to the `Random.Default`.
So changing its value doesn't fail the test with persistent deserialization, and on the opposite, adding a public method to `Random.Default` fails the test, which indicates that `serialVersionUID` for this class is still automatic.

Given the above I think we have no other way than to use a serialization surrogate.

Reply:
What do you mean by serialization surrogate?

Reply:
Clear

Reply:
Can be a nested class in `Random` or `Random.Default`